File: yukicoder/481_1から10/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        for (int i = 1; i <= 9; i++) {

            int num = sc.nextInt();

            if (num != i) {
                System.out.println(i);
                return;
            }

        }

        System.out.println(10);

    }

}

File: yukicoder/481_1から10/Main.cpp
#include <iostream>

using namespace std;

int main() {

    for (int i = 1; i <= 9; i++) {

        int num;
        cin >> num;

        if (num != i) {
            cout << i << endl;
            return 0;
        }

    }

    cout << 10 << endl;

    return 0;

}
